CRM/Contact/BAO/ContactType.php

```diff
@@ -376,7 +376,7 @@ public static function contactTypePairs($all = FALSE, $typeName = NULL, $delimit
   public  static function getSelectElements(
     $all = FALSE,
     $isSeparator = TRUE,
-    $separator = '__'
+    $separator = '--'
```

Comment:
Why are you changing the default value for this function? Shouldn't it be sufficient to change the value passed in when calling the function?

Reply:
Hi @colemanw ,

I changed here because the code is using same separator at https://github.com/civicrm/civicrm-core/pull/12416/files#diff-e54381bfdf51e31cab376c71ca0d66ffR1615 

I tried to update only the separator changes from one of your [commit](https://github.com/civicrm/civicrm-core/commit/46b3417a0).